integration/db/src/main/resources/db.yml

```diff
@@ -0,0 +1,154 @@
+spring:
+  jpa:
+    open-in-view: false
+    hibernate:
+      ddl-auto: none
+    properties:
+      hibernate.default_batch_fetch_size: 100
+
+---
+spring.config.activate.on-profile: local
+
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: create
```

Comment:
This may cause conflicts with the Liquibase integrations.

Reply:
Use `spring.liquibase.enabled` to enable Liquibase migration’s execution on startup